### PR TITLE
refactoring main

### PR DESCRIPTION
**Pull request by alexsukhrin**

### CODE REVIEW

File: src/main/scala/palefat/data/mirror/Runner.scala [hunk 1]
```diff
@@ -27,60 +27,51 @@ import wvlet.log.LogSupport
 
 object Runner extends LogSupport {
 
-  def main(args: Array[String]): Unit = {
-
+  def initConfig(args: Array[String]): Config = {
     val config: Config = ConfigBuilder.build(ConfigBuilder.parse(args))
     logger.debug(s"Parameters parsed: ${config.toString}")
     logger.info(
```
Comment (VShkaberda) [inline src/main/scala/palefat/data/mirror/Runner.scala]: It should be inside `setSparkContext` method. Or returned to `main` and called directly before `setSparkContext`.

**Review by VShkaberda — CHANGES_REQUESTED** [review]

File: src/main/scala/palefat/data/mirror/Runner.scala (hunk 1 above)
Comment (alexsukhrin) [reply src/main/scala/palefat/data/mirror/Runner.scala]: https://github.com/Temabit-FOZZY-Group/mirroring/pull/6/files#diff-d883aa561b4ad6c3252dfa4d282256da5e038b8551735e62da70c4386f3a6787R47
it is called before setSparkContext

Comment (VShkaberda) [reply src/main/scala/palefat/data/mirror/Runner.scala]: Setting spark context in `setSparkContext` feels much more related to getting/creating spark session unlike `initConfig`. But it looks like spark session is obtained in Config, in `FlowLogger.init`, so this logging message is far from "Creating spark session" either way. It seems better to alter it into
```scala
logger.info(
    s"""Spark session is created with configuration: ${spark.conf.getAll
        .mkString(", ")}"""
    )
```

Comment (alexsukhrin) [reply src/main/scala/palefat/data/mirror/Runner.scala]: did not immediately understand you,
moved the logging string to the spark method

**Review by VShkaberda — APPROVED** [review]

**Review by hanna-liashchuk — APPROVED** [review]

**Pull request merged**